py/htm/encoders/grid_cell_encoder.py

```diff
@@ -60,14 +61,14 @@ def __init__(self,
         assert(self.sparsity <= 1)
 
         # Assign each module a range of cells in the output SDR.
-        partitions       = np.linspace(0, self.size, num=len(self.periods) + 1)
+        partitions       = np.linspace(0, super().size, num=len(self.periods) + 1)
```

Comment:
subclass has all the members,methods of its superclass, so this could/should remain `self.size`, or even `size`